frontend/src/metadata/popups/SliceFinderPopup.svelte

```diff
@@ -172,85 +182,132 @@
 					placeholder="Alpha" />
 			</div>
 			<div>
-				<div class="options-header">Min. Support</div>
+				<div style="display:flex">
+					<div class="options-header">Min. Support</div>
+					<div
+						class="information-tooltip"
+						style="margin-top: 3px;"
+						use:tooltip={{
+							content: "controls the minimum slice threshold",
+							theme: "zeno-tooltip",
+							maxWidth: "270",
+						}}>
+						<Icon style="outline:none" component={Svg} viewBox="-6 -6 36 36">
+							<path d={mdiInformationOutline} />
+						</Icon>
+					</div>
+				</div>
 				<Svelecte
-					style="margin-right: 5px; width: 100px"
+					style="margin-right: 5px; width: 120px"
 					bind:value={minimumSuppIdx}
 					options={minimumSupps}
 					placeholder="Minimum Support Threshold" />
 			</div>
 			<div>
-				<div class="options-header">Order By</div>
+				<div style="display:flex">
+					<div class="options-header">Order By</div>
+					<div
+						class="information-tooltip"
+						style="margin-top: 3px;"
+						use:tooltip={{
+							content:
+								"Ordered by cumulative slice score obtained through the sliceline function",
```

Comment:
maybe `Order by slice score, a combination of size and metric`

Reply:
done